DEPLOY.md

```
@@ -370,10 +370,10 @@ Server (NixOS on DigitalOcean)
 
 ## Environment variables
 
-Set `RUST_LOG` to control log verbosity (configured in `.env`):
+Set `RUST_LOG` to control log verbosity. The deployed systemd service sets this
+in `os.nix`:
 ```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)